core/sr-primitives/Cargo.toml

```diff
@@ -14,6 +14,7 @@ substrate-primitives = { path = "../primitives", default-features = false }
 rstd = { package = "sr-std", path = "../sr-std", default-features = false }
 runtime_io = { package = "sr-io", path = "../sr-io", default-features = false }
 log = { version = "0.4", optional = true }
+keyring = { package = "substrate-keyring", path = "../keyring", optional = true }
```

Comment:
What is keyring needed for?

Reply:
This grumble is unaddressed

Reply:
I think this doesn't matter since this one is being superseded by #2048